apps/finance/app/src/components/Transfers.js

```diff
@@ -205,7 +202,7 @@ class Transfers extends React.PureComponent {
                         key={transfer.transactionHash}
                         token={tokenDetails[toChecksumAddress(transfer.token)]}
                         transaction={transfer}
-                        wideMode={width > WIDE_MODE_FROM}
+                        smallViewMode={below('medium')}
```

Comment:
cc @AquiGorka on this, just to be sure this is safe 😄

Reply:
Oh yeah these two props seem similar but are unrelated:

- `wideMode` is an old one, that was used to make the badge shortened / not shortened (it was the first use of what became `Viewport`), and we don’t need it anymore since the badge should always be shortened now.
- `smallViewMode` is the small view mode that @AquiGorka introduced: I made it as an external prop to not have unnecessary instances of `Viewport`.

Reply:
What @bpierre said is half-correct: `wideMode` was used to set the shorten property for the `IdentityBadge` which is lost [now](https://github.com/aragon/aragon-apps/pull/671/files#diff-160b40bf78932dbc027b7f8d9567d1a8R79)? This might break spacing for mobile, should it be set to `true` directly as the default mode?